core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -198,6 +198,11 @@ public interface NativeConfig {
      */
     Optional<Boolean> containerBuild();
 
+    /**
+     * Explicit configuration option to generate PIE native binaries.
```

Comment:
I'm not sure I understand when PIE will be enabled, especially when this is not set. I think we should make it more explicit in the doc.